db/column_family.cc

```diff
@@ -984,11 +984,12 @@ Compaction* ColumnFamilyData::CompactRange(
     const MutableCFOptions& mutable_cf_options, int input_level,
     int output_level, uint32_t output_path_id, uint32_t max_subcompactions,
     const InternalKey* begin, const InternalKey* end,
-    InternalKey** compaction_end, bool* conflict) {
+    InternalKey** compaction_end, bool* conflict,
+    uint64_t max_sst_file_number) {
```

Comment:
It's the same problem as many other places: the name `max_sst_file_number` is hard to make sense. I suggest we pick a better argument name here. Also, document what it means in comments.